### ISSUE TITLE
lint warnings from {Redux,React}TestComponent.js

### ISSUE THREAD

**Issue by jboxman** [bonham000/fcc-react-tests-module#2]
Would it make sense to set an ignore for these?

```
Warning in ./src/test-components/ReactTestComponent.js

/Users/jasonb/Self/projects/fcc-react-tests-module/src/test-components/ReactTestComponent.js
   97:41   warning  There should be no space before '='  react/jsx-equals-spacing
   97:41   warning  There should be no space after '='   react/jsx-equals-spacing
   97:58   warning  There should be no space after '='   react/jsx-equals-spacing
...
✖ 80 problems (0 errors, 80 warnings)
```

**Comment by bonham000** [comment]
Yes, that would be great! I don't know how to do that at the moment but that would be great.

Also just to note if you navigate between challenges React will probably throw errors in the console about calling setState on unmounted components but I think these can be safely ignored.

**Comment by no-stack-dub-sack** [comment]
@jboxman @bonham000 can we ignore all of one type of warning with one of these?
`Use // eslint-disable-next-line to ignore the next line.`
`Use /* eslint-disable */ to ignore all warnings in a file.`

Not sure where exactly to put it.... but i guess we wouldn't want to ignore all warnings

**Comment by jboxman** [comment]
If there are no objections, the easiest way to fix it is to remove all the leading and trailing spaces before the = in the JSX. That will make it the linter happy and no rules need to be disabled.

**Comment by no-stack-dub-sack** [comment]
@jboxman @bonham000 I suppose I would agree with that, but Sean that's up to you. I tend to prefer the look of it without the trailing spaces anyway 😄

**Comment by bonham000** [comment]
Yeah I guess I got into the habit of liking the spaces. Okay I tried to remove as many spaces as I could find but `eslint` is still throwing a ton of errors for the use of `eval` all of the place so I just went through and added `/* eslint-disable */` to pretty much all the `.js` files so now the console is clear and should be easier to work on the tests by console.logging everything.

And I've just pushed all the updated changes to GitHub.
